resources/usr/bin/openhab-cli

```diff
@@ -121,6 +123,42 @@ case "$option" in
     tail -f "${OPENHAB_LOGDIR:?}"/*.log
     ;;
 
+  "info")
+    openHABVersion="$(awk '/openhab-distro/{print $3}' "$OPENHAB_USERDATA/etc/version.properties")"
+    if test "${openHABVersion#*SNAPSHOT}" != "SNAPSHOT"; then
+      buildNumber="($(awk '/build-no/{print $4}' "$OPENHAB_USERDATA/etc/version.properties"))"
+    fi
```

Comment:
Side note: We should synchronize our efforts here and in the existing openHABian code.
Over there it's however a bit easier because everything is developed with bash compatibility...

Reply:
Yeah, I didn't realise that you'd use `sed` for this which would have been equally applicable!